<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cfgmanager</class>
 <widget class="QMainWindow" name="cfgmanager">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>488</width>
    <height>283</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>cfgmanager</string>
  </property>
  <widget class="QWidget" name="main_widget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="1">
     <widget class="QLabel" name="cfg_name_label">
      <property name="text">
       <string>cfg name</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLineEdit" name="cfg_name"/>
    </item>
    <item row="2" column="0" rowspan="6" colspan="4">
     <widget class="QTextEdit" name="text_area"/>
    </item>
    <item row="1" column="4">
     <widget class="QPushButton" name="save_button">
      <property name="text">
       <string>Save cfg</string>
      </property>
     </widget>
    </item>
    <item row="4" column="4">
     <widget class="QPushButton" name="loop_button">
      <property name="text">
       <string>Loop cfg</string>
      </property>
     </widget>
    </item>
    <item row="5" column="4">
     <widget class="QPushButton" name="load_button">
      <property name="text">
       <string>Load cfg</string>
      </property>
     </widget>
    </item>
    <item row="3" column="4">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="7" column="4">
     <widget class="QPushButton" name="quit_button">
      <property name="text">
       <string>Quit</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="2">
     <widget class="QLineEdit" name="location_name"/>
    </item>
    <item row="0" column="4">
     <widget class="QPushButton" name="location_button">
      <property name="text">
       <string>Location</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="cfg_location_label">
      <property name="text">
       <string>cfg location</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
